Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/ui/upload/history/HistoryItemAdapter.kt

```diff
@@ -61,6 +61,6 @@ class HistoryItemAdapter : BaseAdapter<HistoryItemAdapter.CachedKeyViewHolder>()
     }
 
     companion object {
-        private val FORMATTER = DateTimeFormatter.ofPattern("yyyy.MM.dd - HH:mm:ss")
+        private val FORMATTER = DateTimeFormatter.ofPattern("MM.dd.yyyy - HH:mm:ss")
```

Comment:
1. Bye bye, superior ISO 8601 😢 
2. In the ticket, they expect the EU date format with dd.MM.yyyy. Pls no US format, this gives me headaches 😄

Reply:
Thanks for spotting it 😃